lib/site-utils.js

```diff
@@ -63,6 +64,8 @@ const EURO_SYMBOL_PREFIX = '€';
 const YEN_TEXT_PREFIX = '円';
 const YEN_SYMBOL_PREFIX = '￥';
 const GBP_SYMBOL_PREFIX = '£';
+const GBP_TEXT_PREFIX = 'gbp';
+const INR_SYMBOL_PREFIX = 'r ';
```

Comment:
I'm sorry if I'm not understanding, but why does the `INR_SYMBOL_PREFIX` need a space in it?  Also, doesn't the case matter in both these instances (`gbp` and `r`)?

I tried out an example:
```
_.includes("R 4,555", 'r');   // false
_.includes("R 4,555", 'r ');  // false
_.includes("R 4,555", 'R');   // true
```
But this doesn't handle the case that the text is in a different case:
```
_.includes("R 4,555", 'R');  // true
_.includes("r 4,555", 'R');  // false
```
So I would think we'd need to do something like this:
```
_.includes("R 4,555".toUpperCase(), 'R');  // true
_.includes("r 4,555".toUpperCase(), 'R');  // false
```
Which would mean in our code we would have:
```
const GBP_TEXT_PREFIX = 'GBP';
const INR_SYMBOL_PREFIX = 'R';
```
and
```
_.includes(priceString.toUpperCase(), GBP_TEXT_PREFIX)

_.includes(priceString.toUpperCase(), INR_SYMBOL_PREFIX)
```